pkg/tests/apis/alerting/notifications/receivers/receiver_test.go

```diff
@@ -481,6 +481,14 @@ func TestIntegrationAccessControl(t *testing.T) {
 		)...),
 	})
 
+	// Test receivers with uids longer than 40 characters. User name is used in receiver name.
+	adminLikeUserLongName := helper.CreateUser("adminLikeUserCreatingAReallyLongReceiverName", apis.Org1, org.RoleNone, []resourcepermissions.SetResourcePermissionCommand{
+		createWildcardPermission(append(
+			[]string{accesscontrol.ActionAlertingReceiversCreate},
+			ossaccesscontrol.ReceiversAdminActions...,
+		)...),
+	})
```

Comment:
This does not seem to be used anywhere. It looks unfinished. Perhaps just remove it? Or write a separate test?

Reply:
Discussed. The user name is used as part of the receiver name during test case running.